db/column_family.cc

```diff
@@ -616,7 +619,7 @@ void ColumnFamilyData::RecalculateWriteStallConditions(
                imm()->NumNotFlushed() >=
                    mutable_cf_options.max_write_buffer_number - 1) {
       write_controller_token_ =
-          SetupDelay(ioptions_.delayed_write_rate, write_controller,
+          SetupDelay(mutable_db_options_.delayed_write_rate, write_controller,
```

Comment:
I guess when we use `SetDBOptions` to update `max_delayed_write_rate`, we update `delayed_write_rate` to be equal to `max_delayed_write_rate` as well?